hydroengine_service/datasets/dataset_visualization_parameters.json

```diff
@@ -218,5 +218,75 @@
 			"wind_speed": ["#172313", "#144b2a", "#187328", "#5f920c", "#aaac20", "#e1cd73", "#fffdcd"],
 			"wind_direction": ["#a8780d", "#d74957", "#d02fd0", "#7d73f0", "#1e93a8", "#359943", "#a8780d"]
         }
-	}
+	},
+    "projects/dgds-gee/gtsm/waterlevel_return_period": {
+        "type": "Image",
+		"bandNames": {
```

Comment:
when would you ever need this, if you already use the 'name' to look up the 'name'?

Reply:
true, not needed explicitly here. But architecture needs to change a bit to allow this